packages/d/dj-lightbox.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "dj-lightbox",
+  "filename": "js/dj-lightbox.min.js",
+  "description": "DJ Lightbox is small javascript library used to overlay images on top of the current page. It's a snap to setup and works on all modern browsers.",
+  "homepage": "https://github.com/danyg790/dj-lightbox",
+  "keywords": [
+    "lightbox",
+    "maximize",
+    "overlay",
+    "gallery",
+    "slideshow",
+    "images"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/danyg790/dj-lightbox.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Please avoid using glob stars, as these can lead to files being included in the CDN that were not intended. You should ideally be specifying wildcards with extensions, for example `*.js`, or `*.@(css|js)`

Reply:
I have made the change to eliminate the glob stars and set it to include only the js and css files